src/cookbooks/looker/performance_caching.md

```diff
@@ -0,0 +1,75 @@
+# Optimizing Looker Performance - Caching
```

Comment:
This is a great guide. Since the pages in thie cookbooks are related to specific Mozilla implementations and this is a generic guide, I wonder if it fits better in the [Looker Udemy course](https://mozilla.udemy.com/instructor/course/4606482/manage/curriculum), probably as a written guide instead of a video?

Reply:
The auto-generated part is pretty specific to our lookml-generator implementation but this could be cross-posted there for sure